### PR TITLE
[RHPAM-2576] Install mariadb and postgres jdbc drivers

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

Fix https://issues.redhat.com/browse/RHPAM-2576

### CODE REVIEW

**Comment by ruromero** [comment]
@spolti can you review? Thanks

File: jboss-kie-process-migration/module.yaml [hunk 1]
```diff
@@ -4,3 +4,11 @@ version: '1.0'
 description: The jboss-kie-process-migration module.
 execute:
     - script: configure.sh
+artifacts:
```
Comment (spolti) [inline jboss-kie-process-migration/module.yaml]: are these packages provided by Red Hat?

wouldn't be better installs it via microdnf/yum and them just do a symlink to the expected location?

Comment (ruromero) [reply jboss-kie-process-migration/module.yaml]: I think it has to be installed either by fetching the artifacts from maven repo or from the cekit cacher. Do you know a repo I could add to the content_sets.yaml?

**Comment by ruromero** [comment]
@spolti I haven't been able to do a build. Can you help me with that? Theoretically the changes I made should be enough to get what you requested. Thanks

**Comment by spolti** [comment]
Yes, the changes looks to be good, which issue are you facing?

**Comment by ruromero** [comment]
@spolti the build works locally with the only issue of the `nodocs` which I don't know if would affect the productised builds

**Comment by spolti** [comment]
@ruromero can you try this image instead registry.redhat.io/ubi8/ubi-minimal:8.0?
With this image I didn't see the nodocs issue while using microdnf.

**Comment by ruromero** [comment]
@errantepiphany everything PIM related was postponed to the next release

**Comment by ruromero** [comment]
@errantepiphany @spolti is it good to go for this release?

File: jboss-kie-process-migration/module.yaml (hunk 1 above)
Comment (errantepiphany) [reply jboss-kie-process-migration/module.yaml]: @ruromero , @spolti It looks like these are being installed as packages now... Is this good to go now?

**Pull request merged**